### ISSUE TITLE
BulkExport Fetch Days error code undefined.

### ISSUE THREAD

**Issue by tigerfarm** [twilio/twilio-cli#207]
### Custom BulkExport Job Fetch the Data
My goal is to use BulkExport custom option to generate day by day log data. 
List the data information. 
Fetch day resource link so that I can download the day log data file.

### Steps to Reproduce
1. Create a job
```
$ twilio api:bulkexports:v1:exports:jobs:create \
    --resource-type Messages \
    --friendly-name ExportJuly27b \
    --start-day 2020-07-01 \
    --end-day 2020-07-27 \
    --properties=friendlyName \
    --email dthurston@twilio.com \
    --webhook-method GET \
    --webhook-url http://www.example.com/echo
```
I received neither email notice nor a GET request to my URL. I imagine that's a BulkExport issue not a Twilio CLI issue.

2. List the jobs. 
The first example gives a misleading error message because the default is not valid. 
The second example works fine, once I figured that properties is required, not as the help suggested which was that it was optional.
```
$ twilio api:bulkexports:v1:exports:jobs:list --resource-type Messages
 » "sid" is not a valid property name.
$ twilio api:bulkexports:v1:exports:jobs:list --resource-type Messages --properties=friendlyName,jobSid
Friendly Name  Job SID                           
ExportJuly27   JS1dca2e0dfb7815c1fea2362d9f61c16c
ExportJuly27b  JSed5cd7ba5451574475d4d4bb890aed4f
```
3. Given the above, I confirm the job is complete. 
Took me time to figure out the properties, since this had the same misleading error message.
```
$ twilio api:bulkexports:v1:exports:jobs:fetch --job-sid=JSed5cd7ba5451574475d4d4bb890aed4f --properties=friendlyName,startDay,endDay,details
Friendly Name  Start Day   End Day     Details                                            
ExportJuly27b  2020-07-01  2020-07-27  {"0":{"status":"Completed","count":27,"days":null}}
```
4. List day reports to get a list of the day-date values. I'll use, "day": "2020-07-26", in the following step. 
```
$ twilio api:bulkexports:v1:exports:days:fetch --resource-type=Messages --day= --properties=days
[{"friendly_name":"ExportJuly27b","create_date":"2020-07-27","day":"2020-07-01","resource_type":null,"size":750},
...
{"friendly_name":"ExportJuly27b","create_date":"2020-07-27","day":"2020-07-26","resource_type":null,"size":398}
]
```
5. Fetch the job's information returns error code. 
Note, I tried without "=", which made no difference. I tried various other options, none worked for me.
```
$ twilio api:bulkexports:v1:exports:days:fetch --resource-type=Messages --day=2020-07-26
 » Error code undefined from Twilio: undefined. See undefined for more info.
```
This is where I need the help, thank you.
Note, the curl command works.
```
curl -X GET 'https://bulkexports.twilio.com/v1/Exports/Messages/Days/2020-07-26' \
-u TwilioAccountSID:TwilioAuthToken

{"redirect_to": "https://...s3.amazonaws.com/daily/day%3D2020-07-26/type%3DMessages/account%3DACa...3/..."}
```

**Comment by childish-sambino** [comment]
1. Yup, that's likely a BulkExport issue.
2. Duplicate of https://github.com/twilio/twilio-cli/issues/186, see my comment there for further tracking.
3. Yup, best to just use `-o json` to see all the properties in cases like this.

For the last one, I'm still waiting on a bulkexport job to complete to test out the behavior ...

**Comment by childish-sambino** [comment]
Interesting. So this is basically a new use case which the CLI is not handling properly. The `GET` returns a `307` but the CLI is set to not follow redirects automatically. Enabling the redirect results in AWS complaining about there being multiple auth mechanisms:

```
<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>InvalidArgument</Code>
  <Message>Only one auth mechanism allowed; only the X-Amz-Algorithm query parameter, Signature query string parameter or the Authorization header should be specified</Message>
  <ArgumentName>Authorization</ArgumentName>
  <ArgumentValue>Basic blurgh</ArgumentValue>
  <RequestId>1234567890</RequestId>
  <HostId>boo</HostId>
</Error>
```

So, this is a bug, but what should be the expected behavior? Output the S3 URL like cURL is doing? Makes it a pretty generic solution, but what are some other options?

**Comment by tigerfarm** [comment]
```
$ twilio api:bulkexports:v1:exports:days:fetch --resource-type=Messages --day=2020-07-26
 ...
```
Outputting the S3 URL would be functional as long as a person doesn't mind using a browser or another program to download the file. Personally, I'd like the command to download the text file to screen or to a file. I don't see the reasoning for another step to actually download the data. That was what I was expecting when I ran the curl command.

**Comment by childish-sambino** [comment]
Going with the solution to output the S3 URL. As discussed internally, a BulkExports plugin would be better suited to handle any additional functionality.
